<ui version="4.0" >
 <class>Form</class>
 <widget class="QMainWindow" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Motor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Team Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="teamName" >
        <property name="font" >
         <font>
          <family>Bitstream Vera Serif</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="autoFillBackground" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Team 1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" >
     <widget class="QPushButton" name="printScreen" >
      <property name="text" >
       <string>Print Screen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QFrame" name="pFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>384</width>
          <height>256</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="vFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>384</width>
          <height>256</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" >
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Motion Parameters</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>4</number>
         </property>
         <property name="topMargin" >
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Distance (mm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="D" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>115</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text" >
            <string>100.0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Velocity (mm/s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="V" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Acceleration Time (s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TA" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Proportional Gain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="P" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Integral Gain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="I" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QPushButton" name="go" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset>button_go.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>100</width>
              <height>75</height>
             </size>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset>button_stop.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>81</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="defaults" >
            <property name="text" >
             <string>Reset to Defaults</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Results</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>4</number>
         </property>
         <property name="topMargin" >
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Av. Position Error (mm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="error" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>115</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Total Move Time (s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="time" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>100</red>
                 <green>255</green>
                 <blue>44</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>115</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2" >
     <widget class="QProgressBar" name="progressBar" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="value" >
       <number>100</number>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
